### PR TITLE
Install s3cmd and packages as part of separate recipe in core

### PR DESCRIPTION
**Pull request by anguss00**
Together with https://github.com/aodn/chef-private/pull/2609

### CODE REVIEW

File: cookbooks/imos_backup/recipes/default.rb [hunk 1]
```diff
@@ -113,6 +113,13 @@
     mock_credentials = false
   end
 
+  # AS: make sure we're using the python s3cmd utility
```
Comment (lwgordonimos) [inline cookbooks/imos_backup/recipes/default.rb]: Not sure we need a comment here? Installing the package is pretty readable and self explanatory. Coming back in 6 months time to comments like this is more confusing that helpful!

File: cookbooks/imos_po/recipes/s3_credentials.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #
 # Cookbook Name:: imos_po
-# Recipe:: s3cmd
+# Recipe:: s3_credentials
```
Comment (lwgordonimos) [inline cookbooks/imos_po/recipes/s3_credentials.rb]: "s3_credentials" is a bit ambigious. This is specific to s3cmd, so we should be careful not to imply that it is related to any other official S3 config (e.g. boto/awscli). So I'd maybe go for "s3cmd_credentials"?

**Comment by anguss00** [comment]
Changes up @lwgordonimos, also in private

File: cookbooks/imos_core/recipes/s3_cmd.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#
+# Cookbook Name:: imos_core
+# Recipe:: s3_cmd
```
Comment (lwgordonimos) [inline cookbooks/imos_core/recipes/s3_cmd.rb]: One last final thing... can we call this "s3cmd" so it matches the name of the utility, i.e. without the underscore?

**Comment by anguss00** [comment]
Doneskis @lwgordonimos

**Pull request merged**